code/game/objects/items.dm

```diff
@@ -235,12 +237,17 @@ var/global/image/fire_overlay = image("icon" = 'icons/effects/fire.dmi', "icon_s
 
 		msg += "[force_str] melee damage in its current state. [throwforce_str] throwing damage in its current state."

```

Comment:
Yeah, not sure how to modify the capitalisation of a var though. That was what I did first, and it results in a capital letter in the middle of the sentence.